MKMProvider/.gitignore

```diff
@@ -1,2 +1,4 @@
-vendor/
-phpunit.phar
+node_modules
+yarn.lock
+data
+expansion.json
```

Comment:
`expansion.json` is usless, because it should be persist in `data` directory as specified in https://github.com/ImprontaAdvance/telegram-mtg-bot/blob/ef45bb9971108e9725b157ea9f525c91767fa776/MKMProvider/tools/makesets.js#L9